sda-commons-metadata-context/build.gradle

```
@@ -1,15 +1,8 @@
 dependencies {
-  api 'org.springframework.cloud:spring-cloud-starter-openfeign'
-  api 'org.springframework.boot:spring-boot-starter-web'
   api 'org.apache.commons:commons-lang3'
-  api 'javax.ws.rs:javax.ws.rs-api'
-  api 'org.glassfish.jersey.core:jersey-common', {
-    exclude group: 'jakarta.ws.rs', module: 'jakarta.ws.rs-api'
-  }
 
   // overall testing dependencies
-  testImplementation project(':sda-commons-web-testing')
-  testImplementation project(':sda-commons-web-autoconfigure')
+  testImplementation 'org.springframework.boot:spring-boot-starter-test'
```

I think, we can simplify the test dependencies even more:

```suggestion
  testImplementation 'org.junit.jupiter:junit-jupiter'
  testImplementation 'org.assertj:assertj-core'
```